src/ophys_etl/workflows/db/schemas.py

```diff
@@ -124,6 +124,8 @@ class OphysROI(SQLModel, table=True):
     width: int
     height: int
     is_in_motion_border: bool  # Set at by segmentation
+    empty_roi_mask: bool = False  # Set at trace extraction
+    empty_neuropil_mask: bool = False  # Set at trace extraction
```

Comment:
Again, I don't think this should be the responsibility of trace extraction to to flag neuropil masks. Could the neuropil masks instead be calculated in the `segment_postprocess` step, and any roi with a neuropil mask with 0 area filtered out? This would require that we associate each neuropil mask with its corresponding roi. @morriscb what do you think?

Reply:
This one's a bit tougher. Currently trace extraction is where the neuropil mask is calculated and stored with the rest of the data from trace extracting. Honestly, this could represent a real failure mode of the pipeline where there are so many ROIs that a valid cell gets dropped due to having no neuropil. I agree with Mike that doing this is a larger change and would be it's own ticket at minimum if we take it up.